config/environments/test.rb

```diff
@@ -21,12 +22,17 @@
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.cache_store = :null_store
```

Comment:
I know we've had cache adventures with tests, is this in fact what we want?

Reply:
...yeah, that is in fact outdated, good catch :turkey: